hydra-node/src/Hydra/API/WSServer.hs

```diff
@@ -87,7 +88,7 @@ wsApp party tracer history callback headStatusP headIdP snapshotUtxoP responseCh
   -- important to make sure the latest configured 'party' is reaching the
   -- client.
   forwardGreetingOnly con = do
-    seq <- atomically $ nextSequenceNumber history
```

Comment:
NOTE: We don't need to output `TimeServerOutput` here. We could just send out `Greetings` type.